File: src/main/java/net/tianben/advancedcombat/AdvancedCombatClient.java
package net.tianben.advancedcombat;

import net.fabricmc.api.ClientModInitializer;
import net.tianben.advancedcombat.util.ModModelPredicateProvider;

public class AdvancedCombatClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		ModModelPredicateProvider.registerModModels();
		ModModelPredicateProvider.registerModItemModelPredicates();
	}
}

File: src/main/kotlin/net/tianben/advancedcombat/AdvancedCombatClient.kt
package net.tianben.advancedcombat

import net.fabricmc.api.ClientModInitializer
import net.tianben.advancedcombat.util.ModModelPredicateProvider

class AdvancedCombatClient : ClientModInitializer {

    override fun onInitializeClient() {
        ModModelPredicateProvider.registerModModels()
        ModModelPredicateProvider.registerModItemModelPredicates()
    }
}